### PR TITLE
Responsive Manager

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

**Comment by mistermendez** [comment]
closing until Feb release

File: sample.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 </head>
 
 <body>
-<div id='container' style='width:640px; height:360px;'></div>
+<div id='container'></div>
```
Comment (ksunika) [inline sample.html]: the height of the player is 0 for me when the size is not defined here.

Comment (mistermendez) [inline sample.html]: in `sample.html` point to css in build dir

**Comment by ksunika** [comment]
Looks good to me.

File: scss/skin/_default.scss [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+video.video {
+  width: 100% !important;
+  height: 100% !important;
+  position: static !important;
+  display: block !important;
```
Comment (chibzu) [inline scss/skin/_default.scss]: These are things the VTC should be handling. If some other module tried to hide the video element with display: none,  for example, we'd be totally clobbering it.

Comment (mistermendez) [inline scss/skin/_default.scss]: corresponding VTC update - http://github.services.ooyala.net/Platform/mjolnir/pull/476

Comment (mistermendez) [inline scss/skin/_default.scss]: I'll test them together before removing Sass from html5-skin

**Comment by mistermendez** [comment]
default video element style updated. VTC inline style takes priority over default style.

**Comment by chibzu** [comment]
Looks good.

**Pull request merged**